cernopendata/modules/fixtures/data/docs/cms-guide-luminosity-calculation/cms-guide-luminosity-calculation.md

```diff
@@ -0,0 +1,249 @@
+# How to calculate luminosity for CMS open data
+
+## What is this?
+
+This document provides instruction for how to calculate the luminosity information
+for CMS in the open data environment. This is done using `brilcalc`, which is a command line tool from the CMS Beam Radiation Instrumentation and Luminosity (BRIL) group.
+
+Before you continue, consider that some of the information that you wish to calculate is already available for [2010](http://opendata-dev.web.cern.ch/record/1050), [2011](http://opendata-dev.web.cern.ch/record/1051), and [2012](http://opendata-dev.web.cern.ch/record/1052) data. The information available includes:
+
+* the integrated luminosity for validated runs and luminosity sections
+* the trigger prescales by run and luminosity section
+* the luminosity by luminosity section for the list of validated runs and luminosity sections
+
+## Install the CMS environment
+
+There are two options for installing the CMS environment, using a Virtual
+Machine (VM) or using a Docker container.
+
+### Virtual machine
+
+Follow the instructions [here](http://opendata.cern.ch/docs/cms-virtual-machine-2011) to install the CMS VM and then the instructions for [getting started](http://opendata.cern.ch/docs/cms-getting-started-2011).
+
+Briefly, to get started once the VM is installed, log in to the VM with the user name `cms-opendata`. Once logged in, click on the `CMS Shell` icon on the desktop to open the terminal.
+
+At the terminal prompt, run the command
+
+`
+$ cmsrel CMSSW_5_3_32
+`
+
+and then change directory
+
+`
+$ cd CMSSW_5_3_32/src
+`
+
+ and then finally intialize the CMS environment
+
+ `
+$cmsenv
+ `
+
+Now we are ready to [install the `brilcalc` package.](#install-brilcalc)
+
+### Docker container
+
+Following the instructions [here](http://opendata.cern.ch/docs/cms-guide-docker) we download and install an image
+and run a Docker container using the following command:
+
+`docker run --name opendata-2011 -it cmsopendata/cmssw_5_3_32 /bin/bash`
+
+When completed successfully we should be at the command prompt:
+
+`cmsusr@a17b2a79d067 ~/CMSSW_5_3_32/src $`
+
+Now we are ready to [install the `brilcalc` package.](#install-brilcalc)
+
+## Install brilcalc
+
+### cvmfs
```

Comment:
Just in case, I put some cvmfs instructions for the docker containers.  So they will [be available](http://opendata-dev.cern.ch/docs/cms-guide-docker#accessing-cvmfs-from-a-container).